cocos/audio/audio-source.ts

```diff
@@ -38,6 +38,11 @@ enum AudioSourceEventType {
     ENDED = 'ended',
 }
 
+class OperationInfo {
+    op: string = '';
```

Comment:
It is better to create a new type for operation type as only some operations are valid.